### DISCUSSION TITLE
[Feature Request] Github Desktop: Add Ability to Delete Branch if it Exists When Creating a New Branch

### DISCUSSION THREAD

**Discussion by nheiting** [community/community#146749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When creating a new branch and there already exists a same branch with the same name, it can be tedious to search the branch and delete it. I'm wonder if there can be some delete option here when it says 'A branch named {{branchName} already exists'? This is for the Github desktop app.

![image](https://github.com/user-attachments/assets/c33a1d41-2a4a-4bef-87c8-dd4b533abd65)

**Comment by MindfulLearner** [comment]
Have you tried in the website? could be a bug!
